tools/validators/instance_validator/validate/handler.py

```diff
@@ -133,13 +133,14 @@ def TelemetryValidationCallback(self, validator):
     Args:
       validator: the telemetry validator that triggered the callback."""
 
+    print('Generating validation report ...')
     report = ''
 
     if not validator.AllEntitiesValidated():
       report += 'No telemetry message was received for the following entities:'
       report += '\n'
       for entity_name in validator.GetUnvalidatedEntities():
-        report += '  ' + entity_name + '\n'
+        report += '  ' + str(entity_name) + '\n'
```

Comment:
is this coercion?  If so, I'd prefer using the explicit method on the object.

Reply:
Can you detail what you mean?
entity_name is of type YAML

Reply:
as far as I can tell it's a string.  https://github.com/google/digitalbuildings/blob/11bd795774d30c2ba10b10d164a7ef280ade2184/tools/validators/instance_validator/validate/handler.py#L47

What I meant by coercion, is that if the thing is a class that has a .name() method, for instance.  I would prefer using the method to get the string rather than depending on the builtin __str__() to get a string value.  Here it appears that there's a different problem.  The GetUnvalidatedEntities() method doesn't actually return entities so its name should really be changed to be something like GetUnvalidatedEntityNames().  Then this would all be clearer.

Reply:
@berkoben Oops, my review took longer than an hour.

Sorry if we overlapped on some comments.

Reply:
updated the name and replaced with ,format